zcash_client_sqlite/src/wallet.rs

```diff
@@ -3642,7 +3642,10 @@ pub(crate) fn queue_transparent_input_retrieval<AccountId>(
         // queue the transparent inputs for enhancement
         queue_tx_retrieval(
             conn,
-            b.vin.iter().map(|txin| *txin.prevout.txid()),
+            b.vin
+                .iter()
+                .map(|txin| *txin.prevout.txid())
+                .filter(|txid| !txid.is_coinbase()),
```

Comment:
Document this as a fix to `WalletDb::transaction_data_requests` ("no longer generates requests for the null (all-zeroes) transaction ID").